packages/sw-cache-expiration/src/lib/plugin.js

```diff
@@ -303,7 +326,9 @@ class Plugin {
    * @param {string} input.cacheName Name of the cache the Responses belong to.
    * @return {Array<string>} A list of the URLs that were expired.
    */
-  async findExtraEntries({cacheName}) {
+  async findExtraEntries({cacheName} = {}) {
```

Comment:
Nit: But this name seems like the opposite of the description: "findExtraEntries" > "Expires entries base on the the maximum cache size."